raw_description/urdf/vacuum_gripper/gripper.urdf.xacro

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro">
+
+  <xacro:include filename="$(find raw_description)/urdf/vacuum_gripper/gripper.gazebo.xacro" />
+
+  <xacro:property name="gripper_length" value="0.14"/>
+  <xacro:property name="gripper_width" value="0.2"/>
+  <xacro:property name="gripper_height" value="0.14"/>
+  <xacro:property name="gripper_body_height" value="0.14"/>
+
+  <xacro:macro name="vacuum_gripper_definition" params="name parent *origin prefix">
+
+    <!-- joint between arm_ee_link and raw_boxgripper_link -->
+    <joint name="${name}_base_joint" type="fixed" >
+        <insert_block name="origin" />
+        <parent link="${parent}" />
+        <child link="${prefix}gripper_frame" />
+    </joint>
+
+    <link name="${prefix}gripper_frame"/>
+
+    <joint name="${prefix}gripper_to_palm" type="fixed">
+        <parent link="${prefix}gripper_frame"/>
+        <child link="${prefix}gripper_body"/>
+        <origin xyz="0.08015 0.0475 0.0" rpy="1.57079632679 0 3.1415"/>
+    </joint>
+
+    <link name="${prefix}gripper_body">
+        <visual>
+            <!--origin xyz="0 0.036 -0.0473" rpy="0 0 0"/-->
+              <!--geometry>
+                <box size="${gripper_length} ${gripper_width} ${gripper_height}"/>
+              </geometry-->
+            <origin xyz="0 0 -0.048" rpy="0 0 0"/>
+            <geometry>
+                <mesh filename="package://raw_description/meshes/vacuum_gripper/Segmentgreifer_2.STL" />
+            </geometry>
+            <material name="silver">
+                <color rgba="0.8 0.8 0.8 1"/>
+            </material>
+        </visual>
+        <collision>
+            <origin xyz="0 0.03 -0.05"/>
+            <geometry>
+                <box size="0.13 0.17 0.11"/>
+            </geometry>
+            <material name="another_blue2">
+                <color rgba="0.38 0.54 0.83 1"/>
+            </material>
+        </collision>
+    </link>
+
+    <joint name="${prefix}gripper_to_tcp" type="fixed">
+        <parent link="${prefix}gripper_frame"/>
+        <child link="gripper"/>
+        <origin xyz=" 0.1263 0.0 0.0" rpy="0 0 0"/>
+    </joint>
+
+    <link name="gripper">
+      <visual>                        
+        <geometry>
+            <sphere radius="0.005"/>
+        </geometry>
+        <material name="white">
+            <color rgba="1 1 1 0.5"/>
+        </material>
+      </visual>
+      <collision>                        
+        <geometry>
+            <sphere radius="0.005"/>
```

Comment:
As far as I remember, MoveIt!/FCL has issues with collision_checking using geometric primitives
Meshes are recommended!
You can use scalable primitive meshes from https://github.com/ipa-bfb/cob_common/blob/a83b14272436efc3cb9d406d084281a444b29323/cob_description/urdf/common.xacro#L6